src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ClientFileMetadataRouteSpec.scala

```diff
@@ -117,6 +153,23 @@ class ClientFileMetadataRouteSpec extends AnyFlatSpec with Matchers with TestReq
     ps.executeUpdate()
   }
 
+  // scalastyle:off magic.number
```

Comment:
Should this be re-enabled after the function, or does that happen automatically?

Reply:
No it needs re-enabling, I've added it back in.